modules.d/95hwdb/module-setup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# This file is part of dracut.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+check() {
+    return 255
+}
+
+# called by dracut
+install() {
+    inst_multiple -o "${udevdir}"/hwdb.bin
+
+    if [[ $hostonly ]]; then
+        inst_multiple -H -o "${udevconfdir}"/hwdb.bin
```

Comment:
Hmm, but this doesn't fix anything, does it? You only install `/etc/udev/hwdb.bin` for `hostonly`, but the downstream issue is for a generic image (installer).

Reply:
You're right, I've set this to Draft because I had to test. I've rewritten it to work correctly now :).